lib/stoplight/notifier/bugsnag.rb

```diff
@@ -0,0 +1,37 @@
+# coding: utf-8
+
+module Stoplight
+  module Notifier
+    # @see Base
+    class Bugsnag < Base
+      DEFAULT_OPTIONS = {
+        severity: 'info'
+      }
+
+      StoplightStatusChange = Class.new(StandardError)
```

Comment:
This should probably subclass [`Stoplight::Error::Base`](https://github.com/orgsync/stoplight/blob/d195eaadb37a085ff79f7bbd594010cf606b8feb/lib/stoplight/error.rb#L5) instead of `StandardError`.